packages/web-runtime/src/store/user.js

```diff
@@ -34,17 +34,23 @@ const actions = {
     // clear oidc client state
     vueAuthInstance.clearLoginState()
   },
-  async logout({ dispatch }) {
-    const logoutFinalizer = (forceRedirect = false) => {
-      // Remove signed in user
-      dispatch('cleanUpLoginState')
-      dispatch('hideModal')
-      dispatch('loadSettingsValues')
+  async logout(context) {
+    const logoutFinalizer = (isOauth2 = false) => {
+      // Force redirect
+      if (isOauth2) {
+        const logoutUrl = context.getters?.configuration?.auth?.logoutUrl
+
+        if (logoutUrl) {
+          return (window.location = logoutUrl)
+        }
 
-      // Force redirect to login
-      if (forceRedirect) {
         router.push({ name: 'login' })
       }
+
+      // Remove signed in user
```

Comment:
Don't we need the store cleanups before the redirect?

Reply:
Well, this is something I can't answer yet, we do a redirect, so I thought the store should be cleaned while accessing the page the next time. 
I moved this code path, as there is already a redirect happening in one of these dispatched actions